testutil/testnode/full_node_test.go

```diff
@@ -90,7 +91,7 @@ func TestIntegrationTestSuite(t *testing.T) {
 func (s *IntegrationTestSuite) Test_FillBlock() {
 	require := s.Require()
 
-	for squareSize := 2; squareSize < appconsts.MaxSquareSize; squareSize *= 2 {
+	for squareSize := 4; squareSize < appconsts.MaxSquareSize; squareSize *= 2 {
```

Comment:
Sorry this wasn't clear in the GH issue or doc comment above `FillBlock` but since this test previously passed with `squareSize := 2` and the FillBlock implementation includes [these lines](https://github.com/celestiaorg/celestia-app/blob/17043f258ee4236c93e5f4e7d3cf3cdd83b4daa9/testutil/testnode/node_interaction_api.go#L172-L173) which appear to indicate that it does support squareSize `2`, proposal to revert the change here, and throw the unsupported square size error when squareSize is `< 2`

```suggestion
	for squareSize := 2; squareSize < appconsts.MaxSquareSize; squareSize *= 2 {
```

Reply:
Changed condition to pass for squareSize >= 2